### PR TITLE
(graphviz) Updated PR with needed updates 

### PR DESCRIPTION
**Pull request by RedBaron2**
@AdmiringWorm @gep13 @pascalberger @majkinetor 
/cc @smattr @magjac 

## Description

This will make the continuing package of `graphviz` be a cmake build as per discussion in #1535 

## Motivation and Context

This will fix issue #1528 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [ ] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](add_link_to_original_repository_location)
- [Open Issues](link_to_the_generic_location_of_open_issues) *Add the different issues underneath, and tick those that are fixed in this PR*
  - [x] Issue 1 #1528 
  - [ ] Issue 2 Link
- [ ] *Include the link to the opened PR that removes the package from the original location*
- [ ] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

**Comment by RedBaron2** [comment]
@AdmiringWorm @majkinetor 
Everything should be in order now for this PR 🤞

File: automatic/graphviz/README.tmp [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-
-The Graphviz layout programs take descriptions of graphs in a simple text language, and make diagrams in useful formats, such as images and SVG for web pages; PDF or Postscript for inclusion in other documents; or display in an interactive graph browser.
-
-Graphviz has many useful features for concrete diagrams, such as options for colors, fonts, tabular node layouts, line styles, hyperlinks, and custom shapes.
+# [<img src="https://cdn.jsdelivr.net/gh/chocolatey/chocolatey-coreteampackages@00f392142cdbdbda147d3cc3ccb1cb593afb996d/icons/graphviz.png" width="48" height="48"/> <PackageName>](https://chocolatey.org/packages/<PackageName>)
+
+** Note **
+
+Starting with version _2.44_ this package contains cmake builds of graphwiz. If you encounter problem with missing executables for the time being continue using the last msbuild version _2.38_
```
Comment (Smattr) [inline automatic/graphviz/README.tmp:5]: I guess it's too late now, but there's a typo here: s/graphwiz/Graphviz

Comment (majkinetor) [reply automatic/graphviz/README.tmp:5]: Fixed. Thx.

Comment (Smattr) [reply automatic/graphviz/README.tmp:5]: While you're in this area, the grammar of the second sentence could be improved with s/encounter problem/encounter problems, and adding a trailing period.

Comment (majkinetor) [reply automatic/graphviz/README.tmp:5]: OK, done, in next release visible tho.

Comment (Smattr) [reply automatic/graphviz/README.tmp:5]: Thanks!

**Pull request merged**